legal_db/static/styles.scss

```diff
@@ -1,23 +1,22 @@
+// Original variables from Vocabulary
```

Comment:
Is this a duplicate of vocabulary variables? is there any special reason for this?
I think the best way to proceed with this is including the variables files directly from Vocabulary in case of a variable update.

Reply:
Yes, here they are duplicated (probably I should remove the word "Original"?), the reason is that the project is using Vocabulary via CDN and SASS/SCSS files [can't be used this way](https://stackoverflow.com/a/16947612/7297376). I wanted to avoid just downloading the Vocabulary files in favor of build a sort of pipeline (with npm and/or some django library to manage this dependency), this way future updates should be easier more manageable, but this needs some research.

Reply:
@krysal do you mind making a ticket for that work so we don't forget it?

Reply:
Created at #73.